server/auth_server.go

```diff
@@ -370,6 +370,22 @@ func updateSettings(saveConfig map[string]map[string]string) error {
 		// authConfig now was prevConfig
 		// saveConfig is to be saved, so authConfig should get added values from saveConfig
 		log.Debugf("Previous saved auth config: %v", authConfig)
+		// saveConfig (updated config) might contain no secret settings, if GetConfig(listOnly=true) was called before it
```

Comment:
I had updated my pr to change the comment around the same time you reviewed...That's why it's making these comments show under outdated comments.